modules/wrapper/rc/options.nix

```diff
@@ -120,35 +126,21 @@ in {
       description = ''
         An attribute set containing global variable values
         for storing vim variables as early as possible. If
-        populated, this soption will set vim variables in the
-        built configRC as the first item.
+        populated, this option will set vim variables in the
+        built luaConfigRC as the first item.
 
-        E.g. {foo = "bar"} will set `g:foo` to "bar" where
+        E.g. {foo = "bar"} will set `vim.g.foo` to "bar" where
         the type of `bar` in the resulting vimscript will be
         infered from the type of the value in the `{name = value}`
         pair.
       '';
     };
 
-    configRC = mkOption {
-      type = oneOf [(dagOf lines) str];
+    pluginRC = mkOption {
+      internal = true;
+      type = either (dagOf lines) str;
       default = {};
-      description = ''
-        Contents of vimrc, either as a string or a DAG.
-
-        If this option is passed as a DAG, it will be resolved
-        according to the DAG resolution rules (e.g. entryBefore
-        or entryAfter) as per the **nvf** extended library.
-      '';
-
-      example = literalMD ''
-        ```vim
-        " Set the tab size to 4 spaces
-        set tabstop=4
-        set shiftwidth=4
-        set expandtab
-        ```
-      '';
+      description = "The internal DAG used to configure plugins";
```

Comment:
Since this is internal, we might want to hide pluginRC from the docs. In which case, pass `visible = false;` as well.

Unless we want people to see it?

Reply:
Up to you, I'm against leaving it visible, since you can always just do `entryAfter ["pluginConfigs"]` and it will end up being a same thing, except without having to look up the plugin's entry name.

Reply:
But `builtConfigRC` already doesn't appear in the manual, isn't `visible = false` implied?